jupyter/nmm-gui.ipynb

```diff
@@ -531,23 +531,30 @@
    "outputs": [],
    "source": [
     "class GameState:\n",
-    "    def __init__(self, state = START_STATE, player = PLAYER_1, computer = False):\n",
+    "    def __init__(self, state = START_STATE, player = PLAYER_1, c1 = None, c1Limit = 1, c2 = None, c2Limit = 1, timeout = None, stepwise = False):\n",
```

Comment:
Wofür ist `c` die Abkürzung?

Reply:
computer :D

Reply:
Ahhh, kind of obvious :D

Reply:
soll ich das ändern?

Reply:
wäre `player_white` und `player_black` nicht etwas eindeutiger?

Reply:
hmm weiß nicht. Dann wäre es ja beim Menschen `player_white = None` 🤔

Wie wäre es mit `ai1` und `ai1Limit`?

Reply:
Hmm, dann weiß man ja immer noch nicht wer von beiden welche Farbe hat. Man kann ja auch Spieler als `PL` bezeichnen und das als default setzten. Wenn dann keins der 3 Typen `MM`, `AB` und `PL` gefunden wird, gibt es einen Fehler

Reply:
In der gui haben wir eigentlich nie weiß und schwarz. Hab alles mit dem Konstanten `PLAYER_1` und `PLAYER_2` bezeichnet ganz oben bei den Deklarationen

Reply:
Hmm, dann `player_1` und `player_2`? Ich finde `ai` trifft es nicht ganz, da die Variable ja auch aussagen kann, dass es Spieler spielt

Reply:
> Hmm, dann weiß man ja immer noch nicht wer von beiden welche Farbe hat. Man kann ja auch Spieler als `PL` bezeichnen und das als default setzten. Wenn dann keins der 3 Typen `MM`, `AB` und `PL` gefunden wird, gibt es einen Fehler

`p1`, `p1Limit` & `p2`, `p2Limit` und dann mit den Werten `PL`, `AB` & `MM`?

Das Attribut `player` gibt es in der Klasse schon und bezeihnet den Spieler, der gerade dran ist

Reply:
Oder von mir aus auch `algorithm_1` und `algorithm_2`. `ai1` sieht kacke aus :P

Reply:
Warte, du hast doch recht. Finde aber `algorithm` beschreibt es besser als `ai`

Reply:
Also `algorithm_1: "MM" | "AB" | None`

Reply:
Um mal ein bisschen Python mir TypeScript zu mischen :D

Reply:
Jap, hab ich so gemacht

So viele neue Zeichen: `+85 −30` Hoffe du bist glücklich D:

Reply:
Ja bin ich <3